cluster/definition.go

```diff
@@ -19,19 +19,26 @@ import (
 	"bytes"
 	"encoding/json"
 	"io"
+	"time"
 
 	ssz "github.com/ferranbt/fastssz"
 	"github.com/libp2p/go-libp2p-core/peer"
 
 	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/z"
```

Comment:
What is this?

Reply:
It is a package that provides support for structured logging and structured errors. For ex:
```
log.Info(ctx, "Charon starting",
		z.Str("version", version.Version),
		z.Str("git_commit_hash", hash),
		z.Str("git_commit_time", timestamp),
	)
```